compatibility/BUILD

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
why do we not hit this via automation? I don’t see a no autocopyright file here.

Reply:
We detect file types (in order to figure out what kind of comment marker to use) based on extension, and this filename does not have an extension.